### DISCUSSION TITLE
removing a tab from the dataset view (struggling to find this).

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7292] [Q&A]
First off - Hope everyone has a nice break over the holiday period. 

Now my problem.  

For my build of ckan, when I am inside a dataset view I can see 3 tabs:   Dataset (active), Groups, Activity Steam - and a button for Manage. 

What I am looking to do is remove Groups as they are not being used.  

I am struggling to find the code which control what tabs are created, any help would be appreciated,  I appreciate the time of this post, please dont rush to this - we can look in January. 

Thanks

**Comment by muhammetcetinyamac** [comment]
Hi,
For tab "ckanext/ibb_theme/templates/snippets/facet_list.html" you can make an improvement here.
